Update CLI docs (#9074)
OpenHands/OpenHands: docs/usage/how-to/cli-mode.mdx

### Running with Python

1. Install OpenHands using pip:
```bash
pip install openhands-ai
```

2. Set the `SANDBOX_VOLUMES` environment variable to specify the directory you want OpenHands to access
([See using SANDBOX_VOLUMES for more info](../runtimes/docker#using-sandbox_volumes)).

3. Launch an interactive OpenHands conversation from the command line:
```bash
openhands
```

<Note>
If you have cloned the repository, you can also run the CLI directly using Poetry:

poetry run python -m openhands.cli.main
</Note>

The first time you run the CLI, it will take you through configuring the required LLM
settings. These will be saved for future sessions.

> The trajectory of the sessions will be saved in `~/.openhands/file_store/sessions`.

Reviewer: ```suggestion
The trajectory of the sessions will be saved in `~/.openhands/sessions`.
```

**Note** - OpenHands requires Python version 3.12 or higher (Python 3.14 is not currently supported)

1. Install OpenHands using pip:
```bash
pip install openhands-ai
```

  Or if you prefer not to manage your own Python environment, you can use `uvx`:

```bash
uvx --python 3.12 --from openhands-ai openhands
```

2. Set the `SANDBOX_VOLUMES` environment variable to specify the directory you want OpenHands to access

> ([See using SANDBOX_VOLUMES for more info](../runtimes/docker#using-sandbox_volumes)).

Reviewer: ```suggestion
```
I'm not sure this one is accurate, actually

SANDBOX_VOLUMES is only used in docker runtime

Reviewer: Maybe leave it out for now, and I think the usual way to run is to expect it to run in the current directory (which it does)

3. Launch an interactive OpenHands conversation from the command line:
```bash
openhands
```

<Note>
  If you have cloned the repository, you can also run the CLI directly using Poetry:

  poetry run python -m openhands.cli.main
</Note>

4. Set your model, API key, and other preferences using the UI (or alternatively environment variables, below).

This command opens an interactive prompt where you can type tasks or commands and get responses from OpenHands.
The first time you run the CLI, it will take you through configuring the required LLM
settings. These will be saved for future sessions.

> The trajectory of the sessions will be saved in `~/.openhands/sessions`.

Reviewer: Same

    python -m openhands.cli.main --override-cli-mode true
```

This launches the CLI in Docker, allowing you to interact with OpenHands.

The `-e SANDBOX_USER_ID=$(id -u)` ensures files created by the agent in your workspace have the correct permissions.

> The trajectory of the sessions will be saved in `~/.openhands-state/sessions`.

Author: Thinking because it's wrong or thinking because that's a bug?

> The trajectory of the sessions will be saved in `~/.openhands-state/sessions`.

Reviewer: I was thinking about [this](https://openhands-ai.slack.com/archives/C08D8FJ5771/p1749928754967519?thread_ts=1749924304.543349&channel=C08D8FJ5771&message_ts=1749928754.967519), please see the slack discussion when you get a chance ❤️

    python -m openhands.cli.main --override-cli-mode true
```

<Note>
  If you used OpenHands before version 0.44, you may want to run `mv ~/.openhands-state ~/.openhands` to migrate your
  conversation history to the new location.
</Note>

This launches the CLI in Docker, allowing you to interact with OpenHands.

The `-e SANDBOX_USER_ID=$(id -u)` ensures files created by the agent in your workspace have the correct permissions.

> The trajectory of the sessions will be saved in `~/.openhands/sessions`.

Reviewer: ```suggestion
The conversation history will be saved in `~/.openhands/sessions`.
```

Just a thought, to keep it simple and, we already use the same words in this document
